src/presentational-components/shared/loader-placeholders.js

```diff
@@ -193,7 +193,7 @@ export const WorkflowLoader = () => {
   const formatMessage = useFormatMessage();
   return (
     <Form>
-      <FormGroup fieldId="2" label="Select workflow">
+      <FormGroup fieldId="2" label="">
```

Comment:
Is the empty label okay?

Reply:
Yes, it's now a common loader. We could customize it but honestly, I don't think that anybody would notice. I would even rather use the spinner.

Reply:
I was speaking more about having an empty prop... according to docs it's not a required attribute.

Reply:
What a feature. Let me remove it.